projects/cdk/pipes/emails/emails.options.ts

```diff
@@ -0,0 +1,18 @@
+import {tuiCreateToken} from '@taiga-ui/cdk/utils/miscellaneous';
+
+export const TUI_EMAIL_PIPE_DEFAULT_OPTIONS = [
+    'mail.ru',
```

Comment:
Let's use international emails by default in public and override them in proprietary providers.

Reply:
fixed